Explorer/Assets/Plugins/TexturesFuse/TexturesServerWrap/Unzips/NodeTexturesFuse.cs

```diff
@@ -19,7 +19,23 @@ internal class NodeTexturesFuse : ITexturesFuse
 
         private const int MEMORY_LIMIT = MB * 1024; //GB
 
-        public const string CHILD_PROCESS = "node.exe";
+        public const string CHILD_PROCESS_NAME = "node.exe";
+        private const string PIPE_PREFIX = "\\\\.\\pipe\\";
+        private const string SPACE = " ";
+        private const string CHILD_PROCESS_COMMAND = CHILD_PROCESS_NAME + SPACE + PIPE_PREFIX + NAMED_PIPE + SPACE + MMF_INPUT + SPACE + MMF_OUTPUT;
+
+        /// <summary>
+        /// Avoid collisions between editor and client
+        /// </summary>
+#if UNITY_EDITOR
+        private const string ENVIRONMENT_POSTFIX = "editor";
+#else
+        private const string ENVIRONMENT_POSTFIX = "client";
+#endif
+
+        private const string MMF_INPUT = "dcl_fuse_i_" + ENVIRONMENT_POSTFIX;
+        private const string MMF_OUTPUT = "dcl_fuse_o_" + ENVIRONMENT_POSTFIX;
+        private const string NAMED_PIPE = "dcl_fuse_p_" + ENVIRONMENT_POSTFIX;
```

Comment:
Can the pipes be named after the process ID or something else that's unique? What if I had 7 copies of the project open in 7 copies of Unity editor running all at the same time?

Reply:
1. You can't run the same unity project by unity's design. (UnityHub would say "Projecr is already open").
2. You can run the same client multiple times, but it doesn't make a sense because you won't get into the multiplayer with the same wallet ID.

And I wonder in which case you theoretically would launch multiple instances?

Reply:
No, multiple copies of the project.

Reply:
Makes a sense, ids would be more reliable